code/_onclick/click.dm

```diff
@@ -327,15 +327,18 @@
 		ML.pulled(src)
 
 /mob/living/CtrlClick(mob/user)
-	if(isliving(user) && Adjacent(user) && !user.incapacitated())
-		if(world.time < user.next_move)
-			return FALSE
-		var/mob/living/user_living = user
-		if(user_living.apply_martial_art(src, null, is_grab=TRUE) == MARTIAL_ATTACK_SUCCESS)
-			user_living.changeNext_move(CLICK_CD_MELEE)
-			return
-	else
-		..()
+	if(!isliving(user) || !Adjacent(user) || user.incapacitated())
```

Comment:
does this mean that you can no longer grab people using extendo-hands? maybe you could check for reach or something?

Reply:
I hope it does, otherwise I don't care enough to fix it.

Reply:
what are you talking about athath this is just an early ..() return nothing functionally is changed